src/main/java/com/example/demo/model/Book.java

```diff
@@ -0,0 +1,87 @@
+package com.example.demo.model;
+
+import java.util.Objects;
+
+public class Book {
+    String title;
+    String author;
```

Comment:
May be also replace variable author of type String  with variable of type Author

Reply:
We already have such a task